package.json

```diff
@@ -102,13 +102,14 @@
     }
   },
   "scripts": {
-    "prebuild": "composer install --no-dev --no-interaction",
-    "build": "wp-scripts build && gulp package",
-    "postbuild": "composer install",
-    "build:zip": "npm run build",
+    "prebuild": "composer install",
+    "build": "wp-scripts build && gulp build && npm run i18n",
+    "build:zip": "npm run build && composer install --no-dev -n && gulp zip",
+    "postbuild:zip": "composer install",
     "dev": "wp-scripts build && gulp build",
     "doc:tracking": "jsdoc ./assets/source/ -c .jsdocrc.json -t ./woo-tracking-jsdoc",
     "format:js": "wp-scripts format ./assets/source",
+    "i18n": "WP_CLI_PHP_ARGS='-d memory_limit=2048M' ./vendor/bin/wp i18n make-pot ./ i18n/languages/$npm_package_name.pot --slug=$npm_package_name --domain=$npm_package_name --include=assets/build,includes,src,views,*.php",
```

Comment:
❓

Is it not a bit too permissive to have *.php? as it will include any PHP file independent of its directory, I see that by default [wp i18n make-pot](https://developer.wordpress.org/cli/commands/i18n/make-pot/) will exclude `node_modules, .git, .svn, .CVS, .hg, vendor, *.min.js.`. but some folders like bin/tests etc could be excluded too. Compared with the GLA plugin we exclude some folders like bin, data, tests, etc

`--exclude=bin,data,js/src,node_modules,tests,vendor`

What do you think?